pkg/kyma/cmd/test/list/cmd.go

```diff
@@ -0,0 +1,90 @@
+package list
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/kyma-project/cli/pkg/kyma/cmd/test"
+	"github.com/kyma-project/cli/pkg/kyma/cmd/test/client"
+	"github.com/kyma-project/cli/pkg/kyma/core"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	opts *options
+	core.Command
+}
+
+func NewCmd(o *options) *cobra.Command {
+	cmd := command{
+		Command: core.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:     "list",
+		Short:   "Show available tests on a running Kyma cluster",
+		Long:    `Show available tests on a running Kyma cluster`,
+		RunE:    func(_ *cobra.Command, _ []string) error { return cmd.Run() },
+		Aliases: []string{"l"},
+	}
+
+	cobraCmd.Flags().BoolVarP(&o.Definitions, "definitions", "d", false, "Show test definitions only")
+	return cobraCmd
+}
+
+func (cmd *command) Run() error {
+	cli, err := client.NewTestRESTClient(10 * time.Second)
+	if err != nil {
+		return fmt.Errorf("unable to create test REST client. E: %s", err)
+	}
+
+	if cmd.opts.Definitions {
```

Comment:
So by default the definitions will not be listed. If you put -d then only the definitions will be listed -> ni mixed mode, right? So really like two separated commands, I think we should have two commands for that.

Reply:
Fixed by introducing a new command `kyma test definitions`